packages/php-wasm/node/src/test/php.spec.ts

```diff
@@ -109,6 +109,44 @@ describe.each(SupportedPHPVersions)('PHP %s', (phpVersion) => {
 		});
 	});
 
+	/**
+	 * @issue https://github.com/WordPress/wordpress-playground/issues/1042
+	 */
+	describe('dns_* functions()', () => {
+		it('dns_check_record should exist and be possible to run', async () => {
+			const result = await php.run({
+				code: `<?php
+				var_dump(dns_check_record('w.org', 2));
+			`,
+			});
+			expect(result.text).toEqual('bool(false)\n');
+		});
+		it('dns_get_record should exist and be possible to run', async () => {
+			const result = await php.run({
+				code: `<?php
+				var_dump(dns_get_record('w.org'));
+			`,
+			});
+			expect(result.text).toEqual('array(0) {\n}\n');
+		});
+		it('dns_get_mx should exist and be possible to run', async () => {
```

Comment:
Should we test `getmxrr` too just to make sure the `ZEND_FALIAS` plumbing is set up correctly?

Reply:
Done in https://github.com/WordPress/wordpress-playground/pull/1067/commits/0a5c53953070039eea92a83827cda84b613372fb